data/json/monsters/mutant_mammal.json

```diff
@@ -427,5 +427,195 @@
+      "DOGFOOD",
+      "CANPLAY",
+      "GROUP_MORALE",
+      "HEARS",
+      "PET_MOUNTABLE",
+      "HIT_AND_RUN",
+      "KEENNOSE",
+      "PATH_AVOID_DANGER_1",
+      "SEES",
+      "SMELLS",
+      "SWARMS",
+      "WARM"
+    ]
+  },
+  {
+    "id": "mon_dog_telekenetic_pup",
+    "type": "MONSTER",
+    "copy-from": "mon_dog_auscattle_pup",
+    "name": { "str": "triclopean puppy" },
+    "description": "A rather cuddly looking pup despite the third eye on its forehead, which has yet to actually open.  Particles of dust from the floor occasionally shift towards it when its excited.",
+    "upgrades": { "age_grow": 42, "into": "mon_dog_telekenetic" }
+  },
+  {
+    "id": "mon_dog_antimemetic",
+    "type": "MONSTER",
+    "name": { "str": "unknowable dog" },
+    "description": "You are completely unable to recall anything about what this dog looks like.  You know it's a dog, you know it's there, you know you can see it, but any further information about it isn't retained in your mind.",
+    "default_faction": "dog",
+    "categories": [ "WILDLIFE" ],
+    "species": [ "MAMMAL" ],
+    "volume": "30000 ml",
+    "weight": "30 kg",
+    "harvest": "mutant_mammal_small_fur",
+    "families": [ "prof_intro_biology" ],
+    "hp": 30,
+    "speed": 150,
+    "material": [ "flesh" ],
+    "symbol": "d",
+    "color": "light_gray",
+    "looks_like": "mon_dog",
+    "aggression": -15,
+    "morale": 15,
+    "aggro_character": false,
+    "melee_skill": 6,
+    "melee_dice": 2,
+    "melee_dice_sides": 3,
+    "melee_damage": [ { "damage_type": "cut", "amount": 3 } ],
+    "dodge": 3,
+    "vision_night": 5,
+    "path_settings": { "max_dist": 10 },
+    "reproduction": { "baby_monster": "mon_dog_antimemetic_pup", "baby_count": 4, "baby_timer": 270 },
+    "//": "1-4 puppies & 270 days per-litter for size small canines",
+    "baby_flags": [ "SPRING", "SUMMER", "AUTUMN", "WINTER" ],
+    "biosignature": { "biosig_item": "feces_dog", "biosig_timer": 6 },
+    "anger_triggers": [ "HURT", "FRIEND_ATTACKED", "FRIEND_DIED" ],
+    "special_attacks": [ [ "EAT_FOOD", 100 ], [ "PARROT_AT_DANGER", 0 ] ],
+    "petfood": {
+      "food": [ "DOGFOOD" ],
+      "feed": "The %s seems to like you!  It lets you pat its head and seems friendly.",
+      "pet": "You pet the %s, you think.  You don't remember what it felt like."
+    },
+    "flags": [
+      "ANIMAL",
+      "DOGFOOD",
+      "CANPLAY",
+      "GROUP_MORALE",
+      "HEARS",
+      "PET_MOUNTABLE",
+      "HIT_AND_RUN",
+      "KEENNOSE",
+      "PATH_AVOID_DANGER_1",
+      "SEES",
+      "SMELLS",
+      "SWARMS",
+      "WARM"
+    ]
+  },
+  {
+    "id": "mon_dog_antimemetic_pup",
+    "type": "MONSTER",
+    "copy-from": "mon_dog_auscattle_pup",
+    "name": { "str": "unknowable puppy" },
+    "description": "Try as you might, your brain is unable to recall any details on what this puppy looks like. All you can remember is that it is a puppy, that it exists, and that it is cute.",
```

Comment:
```suggestion
    "description": "Try as you might, your brain is unable to recall any details on what this puppy looks like.  All you can remember is that it is a puppy, that it exists, and that it is cute.",
```

Reply:
What texture would an unknowable dog even have 🤔

Reply:
grey silhouette of a dog, perhaps?